### PR TITLE
2.9 readme updates

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/17013

I was initially thinking of also updating the readmes in the moped-api and moped-auth directories, but in the interest of getting this updates out, I am going to leave those for a later time. 

## Testing

I welcome any suggestions! I mostly tried to clean up what is no longer applicable. 

(I originally had edits in the main Readme, but removed them in favor of Amenity's PR: https://github.com/cityofaustin/atd-moped/pull/1333) 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved

### CODE REVIEW

**Review by frankhereford — APPROVED** [review]
Thank you so much for this documentation updates in the README 🙏🙏🙏 🚢

File: moped-database/README.md [hunk 1]
```diff
@@ -208,3 +142,5 @@ To version control database views, SQL files are stored in `/views/` and the ste
 More documentation is available in the [MOPED Technical Docs](https://app.gitbook.com/@atd-dts/s/moped-technical-docs/dev-guides/hasura-migrations)
 
 You are also encouraged to learn [Hasura Migrations from their documentation](https://hasura.io/docs/1.0/graphql/core/migrations/index.html). 
+
+[SchemaSpy Analysis of moped](https://db-docs.austinmobility.io/atd-moped-production/index.html)
```
Comment (mddilley) [inline moped-database/README.md:146]: Small thing but do we want Moped capitalized here?

File: moped-database/README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ https://hasura.io/docs/1.0/graphql/core/hasura-cli/install-hasura-cli.html
 #### 3. The Hasura Cluster Helper
 
 There is a tool we've created that has a few shortcuts available, it's called [hasura-cluster](https://github.com/cityofaustin/atd-moped/blob/main/moped-database/hasura-cluster).
-This just a bash script that runs a few docker-compose commands.
```
Comment (mddilley) [inline moped-database/README.md:54]: I don't know why but the old `This just a bash script` typo made me laugh real hard. 😅

**Review by mddilley — APPROVED** [review]
Thanks for making this more accurate and getting us ready for local MUIX dev with the new env template. Eager to look at Frank's PR for the local dev script now. 🚢 🚀

**Pull request merged**